### DISCUSSION TITLE
l5d-proxy-error: HTTP Logical service in fail-fast

### DISCUSSION THREAD

**Discussion by eddielishan** [linkerd/linkerd2#8224] [Q&A]
I found a problem in the test process.Don't know how to reproduce it.   No mesh pod  access is normal

Is it the same question as this question?   https://github.com/linkerd/linkerd2/pull/8169

root@xxxxxx:/# curl http://xxxxx/v2/api-docs -i
HTTP/1.1 504 Gateway Timeout
l5d-proxy-error: HTTP Logical service in fail-fast
connection: close
l5d-proxy-connection: close
content-length: 0
date: Fri, 08 Apr 2022 06:16:58 GMT

linkerx proxy log:
[  9699.764403s]  INFO ThreadId(01) outbound:server{orig_dst=10.233.30.198:8108}:rescue{client.addr=10.233.97.223:54164}: linkerd_app_core::errors::respond: Request failed error=HTTP Logical service in fail-fast

**Comment by olix0r** [comment]
This error generally means that the proxy's load balancer for this service (at `10.233.30.198`) has no available endpoints.
